### PR TITLE
arash/Link to signup page instead of modal

### PR DESCRIPTION
**Pull request by arash-fs**
# Description
use signup page instead of signup modal.
Changes:

-   fixes issue with item
-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Refactor code
-   [ ] Update translation text
-   [ ] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/layout/nav.js [hunk 1]
```diff
@@ -230,19 +228,18 @@ export const Nav = () => {
                         <Button onClick={handleLogin} primary>
                             <span>{localize('Log in')}</span>
                         </Button>
-                        <SignupButton ref={button_ref} secondary onClick={toggleModal}>
-                            <span>{localize('Try for free')}</span>
-                        </SignupButton>
+                        <LocalizedLink to='/signup/'>
```
Comment (oskar-binary) [inline src/components/layout/nav.js]: There is a `LinkButton` that can be used here

File: src/pages/about/_our-locations.js [hunk 2]
```diff
@@ -104,12 +101,11 @@ export const OurLocations = () => {
                         </Location>
                     ))}
                 </OurLocationsContainer>
-                <Button secondary onClick={toggleModal}>
-                    {localize('Start with a free practice account')}
-                </Button>
-                <Modal toggle={toggleModal} is_open={show_modal} closeModal={closeModal}>
-                    <SignupModal autofocus />
-                </Modal>
+                <LocalizedLink to='/signup/'>
```
Comment (oskar-binary) [inline src/pages/about/_our-locations.js]: same here

File: src/pages/about/index.js [hunk 3]
```diff
@@ -52,12 +50,11 @@ const About = () => {
                     <Header as="h1" align="center" lh="1.1">
                         Go ahead, experience it for yourself.
                     </Header>
-                    <AccountButton onClick={toggleModal} secondary>
-                        {localize('Create a free account')}
-                    </AccountButton>
-                    <Modal toggle={toggleModal} is_open={show_modal} closeModal={closeModal}>
-                        <SignupModal autofocus />
-                    </Modal>
+                    <LocalizedLink to='/signup/'>
```
Comment (oskar-binary) [inline src/pages/about/index.js]: and here

File: src/pages/check-email/index.js [hunk 4]
```diff
@@ -33,14 +29,13 @@ const CheckEmail = () => {
                         )}
                     </SectionHeader>
                     <IconGrid />
-                    <Button secondary onClick={toggleModal}>
-                        {localize('Re-enter your email and try again')}
-                    </Button>
+                    <LocalizedLink to='/signup/'>
```
Comment (oskar-binary) [inline src/pages/check-email/index.js]: and here

File: src/pages/keep-safe/index.js [hunk 5]
```diff
@@ -70,9 +68,11 @@ const KeepSafe = () => {
                             'Use our demo account and learn how to trade by using risk-free virtual funds.',
                         )}
                     </StyledHeader>
-                    <Button secondary onClick={toggleModal}>
-                        {localize('Create a free account')}
-                    </Button>
+                    <LocalizedLink to='/signup/'>
```
Comment (oskar-binary) [inline src/pages/keep-safe/index.js]: and here

File: src/pages/home/_why-deriv-mobile.js [hunk 6]
```diff
@@ -98,12 +96,11 @@ const WhyDerivMobile = () => {
                 </CardItem>
             </Card>
             <ButtonWrapper>
-                <SingupButton onClick={toggleModal} secondary>
-                    {localize('Create a free demo account')}
-                </SingupButton>
-                <Modal toggle={toggleModal} is_open={show_modal} closeModal={closeModal}>
-                    <SignupModal autofocus />
-                </Modal>
+                <LocalizedLink to='/signup/'>
```
Comment (oskar-binary) [inline src/pages/home/_why-deriv-mobile.js]: and here

File: src/components/layout/nav.js (hunk 1 above)
Comment (arash-fs) [reply src/components/layout/nav.js]: if I use `LinkButton` here for an unknown reason the calculation of translation gets messy and the login button will not show.

File: src/pages/about/_our-locations.js (hunk 2 above)
Comment (arash-fs) [reply src/pages/about/_our-locations.js]: done

File: src/pages/about/index.js (hunk 3 above)
Comment (arash-fs) [reply src/pages/about/index.js]: done

File: src/pages/check-email/index.js (hunk 4 above)
Comment (arash-fs) [reply src/pages/check-email/index.js]: done

File: src/pages/keep-safe/index.js (hunk 5 above)
Comment (arash-fs) [reply src/pages/keep-safe/index.js]: done

File: src/pages/home/_why-deriv-mobile.js (hunk 6 above)
Comment (arash-fs) [reply src/pages/home/_why-deriv-mobile.js]: done

**Pull request merged**